tests/python/integration/test_lower.py

```diff
@@ -15,41 +15,52 @@
 # specific language governing permissions and limitations
 # under the License.
 # pylint: disable=invalid-name, too-many-locals, too-many-statements, unused-argument
```

Comment:
Is it possible to get rid of some of them now that we are cleaning up tests.

Reply:
Done

>